### PR TITLE
Remove experimental flag for redirects config

### PR DESCRIPTION
**Pull request by matthewp**
## Changes

- Removes the need for the `experimental: { redirects: true }` config.

## Testing

- Tests updated to reflect the change

## Docs

- TBD

### CODE REVIEW

**Review by natemoo-re — APPROVED** [review]
Hooray!

File: .changeset/dry-beers-grow.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+This change moves the `redirects` configuration out of experimental, meaning if you are currently using the:
+
+```js
+experimental: {
+  redirects: true,
+}
+```
+
+You can simply remove this flag. If you have been waiting on using redirects before it stabilized, how you can do so. Checkout [the docs](https://docs.astro.build/en/core-concepts/routing/#redirects) to learn how to use the feature.
```
Comment (sarah11918) [inline .changeset/dry-beers-grow.md:15]:
```suggestion
This change moves the `redirects` configuration out of experimental. If you were previously using experimental redirects, remove the following experimental flag:

```js
experimental: {
  redirects: true,
}
```

If you have been waiting for stabilization before using redirects, now you can do so. Check out [the docs on redirects](https://docs.astro.build/en/core-concepts/routing/#redirects) to learn how to use this built-in feature.
```

Just tightened up some wording, and made it a little more forceful re: what to do now!

File: packages/astro/src/@types/astro.ts [hunk 2]
```diff
@@ -458,7 +457,7 @@ export interface AstroUserConfig {
 	 * @name redirects (Experimental)
```
Comment (sarah11918) [inline packages/astro/src/@types/astro.ts:457]:
```suggestion
	 * @name redirects
```

File: packages/integrations/netlify/README.md [hunk 3]
```diff
@@ -92,7 +92,7 @@ export default defineConfig({
 
 ### Static sites
 
-For static sites you usually don't need an adapter. However, if you use `redirects` configuration (experimental) in your Astro config, the Netlify adapter can be used to translate this to the proper `_redirects` format.
+For static sites you usually don't need an adapter. However, if you use `redirects` configuration in your Astro config, the Netlify adapter can be used to translate this to the proper `_redirects` format.
```
Comment (sarah11918) [inline packages/integrations/netlify/README.md:95]: Nice catch! :raised_hands: 

Just confirming that there isn't something similar for Vercel 's README etc. that should also be removed.

**Review by sarah11918 — COMMENTED** [review]
Love it when a feature graduates from experimental!

Just some tiny things to extra-double-triple check here, then redirects is 2.9 bound! :steam_locomotive:

File: packages/integrations/netlify/README.md (hunk 3 above)
Comment (matthewp) [reply packages/integrations/netlify/README.md:95]: There's not

File: .changeset/dry-beers-grow.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+---
+
+Redirects configuration
+
+This change moves the `redirects` configuration out of experimental. If you were previously using experimental redirects, remove the following experimental flag:
+
+```js
+experimental: {
+  redirects: true,
+}
```
Comment (sarah11918) [inline .changeset/dry-beers-grow.md:12]:
``````suggestion
}
```

If you have been waiting for stabilization before using redirects, now you can do so. Check out [the docs on redirects](https://docs.astro.build/en/core-concepts/routing/#redirects) to learn how to use this built-in feature.
``````
UGH.. nested backticks *almost* won again! *<shakes fist>*

This corrects the commit of a wonkily-formatted suggestion, that cut off some of the intended code.

**Pull request merged**